packages/video_player/tizen/src/video_player.cc

```diff
@@ -205,6 +212,45 @@ VideoPlayer::~VideoPlayer() {
   }
 }
 
+void VideoPlayer::ExecuteSinkEvents() {
+  std::lock_guard<std::mutex> lock(queue_mutex_);
+  while (!encodable_event_queue_.empty()) {
+    if (event_sink_) {
+      event_sink_->Success(encodable_event_queue_.front());
+    }
+    encodable_event_queue_.pop();
+  }
+
+  while (!error_event_queue_.empty()) {
+    if (event_sink_) {
+      event_sink_->Error(error_event_queue_.front().first,
+                         error_event_queue_.front().second);
+    }
+    error_event_queue_.pop();
+  }
+}
+
+void VideoPlayer::PushEvent(flutter::EncodableValue encodable_value) {
+  if (event_sink_ == nullptr) {
```

Comment:
Just for consistency
```suggestion
  if (!event_sink_) {
```

Reply:
Fixed!